packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -45,10 +45,11 @@ export class ZoweVsCodeExtension {
     public static showVsCodeMessage(message: string, severity: MessageSeverityEnum, logger: IZoweLogger): void {
         logger.logImperativeMessage(message, severity);
 
-        const errorMessage = `${logger.getExtensionName()} error: ${message}`;
         if (severity < 3) {
+            const errorMessage = `${logger.getExtensionName()} info: ${message}`;
             void vscode.window.showInformationMessage(errorMessage);
         } else {
+            const errorMessage = `${logger.getExtensionName()} error: ${message}`;
```

Comment:
Since these are pop ups maybe we should include warning in here since we have that option for vsc messages, for instance below 3 is info, 3 is warning, and above 3 is error.  What do you think?

Reply:
and not sure if `info` , `warning`, or `error` is needed in the message itself since the pop up explains the level of information, so proposing just this format
```
const errorMessage = `${logger.getExtensionName()}: ${message}`;
```